libs/lein-template/src/leiningen/new/io/github/kit_clj/options/helpers.clj

```diff
@@ -13,12 +14,22 @@
     options
     {:tag-open \< :tag-close \> :filter-open \< :filter-close \>}))
 
-(def render-text (renderer template-name selmer-renderer))
```

Comment:
I had to override `leiningen.new.templates/renderer` since this has the `leiningen/new` path hard-coded in its implementation.